phpstan.neon

```diff
@@ -0,0 +1,94 @@
+includes:
+	- vendor/nunomaduro/larastan/extension.neon
+	- vendor/phpstan/phpstan-deprecation-rules/rules.neon
+	- vendor/thecodingmachine/phpstan-safe-rule/phpstan-safe-rule.neon
+	- vendor/phpstan/phpstan-strict-rules/rules.neon
+
+services:
+	-
+		class: PHPStan\Rules\StrictCalls\StrictFunctionCallsRule
+		tags:
+			- phpstan.rules.rule
+
+	-
+		class: PHPStan\Rules\SwitchConditions\MatchingTypeInSwitchCaseConditionRule
+		tags:
+			- phpstan.rules.rule
+
```

Comment:
Are you sure that we actually need this? The docs say that this this way individual rules can be enabled if one does not want to include the entire rule set `vendor/phpstan/phpstan-strict-rules/rules.neon` at once. However we include it. Hence, I assume that these lines are not necessary and can be removed.

I also check `vendor/phpstan/phpstan-strict-rules/rules.neon` and `StrictFunctionCallsRule` as well as `MatchingTypeInSwitchCaseConditionRule` are indeed included.

```suggestion
```